Source/WebCore/platform/mock/mediasource/MockMediaPlayerMediaSource.h

```diff
@@ -51,6 +51,7 @@ class MockMediaPlayerMediaSource : public MediaPlayerPrivateInterface, public Ca
 
     void advanceCurrentTime();
     MediaTime currentMediaTime() const override;
+    bool currentMediaTimeMayProgress() const override;
```

Comment:
final

Reply:
there's heaps of them marked as override, I thought it was better to keep the style , even if incorrect